parser_library/src/parsing/grammar/ca_expr_rules.g4

```diff
@@ -12,9 +12,24 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
- //rules for CA expresssions, variable symbol, CA string
+ //rules for CA expressions, variable symbol, CA string
 parser grammar ca_expr_rules; 
 
+expr_general returns [ca_expr_ptr ca_expr]
+	:
+	{NOT(_input->LT(1))}?
+	not_expr_list
+	{
+		$ca_expr = std::move($not_expr_list.ca_expr);
+	}
+	|
+	expr
+	{
+		$ca_expr = std::move($expr.ca_expr);
+	};
```

Comment:
can these rule be merged? Look like the inner "+" rule could be changed to "*" and merged in to this rule?

Reply:
Seems like it 👍